sriovnet_switchdev_test.go

```diff
@@ -432,39 +432,3 @@ func TestGetRepresentorPeerMacAddressDevlink(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "0c:42:a1:de:cf:7c", mac.String())
 }
-
-func TestSetRepresentorPeerMacAddress(t *testing.T) {
```

Comment:
why this test go removed?

Reply:
it should not be removed. This is a mistake.